src/assets/i18n/en.json

```diff
@@ -89,10 +89,87 @@
   "EXACT_MATCH": "Exact match",
   "FAVORITE": "Favorite",
   "FOR_FREE": "For free",
+  "FORMS": {
+    "ABOUT_WORKSHOP": "About workshop",
+    "ADD_TEACHER_LATER": "Teachers can be added to the workshop later",
+    "CONTACTS": "Contacts",
+    "DESCRIPTION": "Description",
+    "TEACHERS": "Teachers",
+    "UAH_FOR": "UAH for",
+    "ACTIONS": {
+      "ADD_ANOTHER_SECTION": "Add another section",
+      "ADD_ANOTHER_TEACHER": "Add another teacher",
+      "ADD_MORE_DAYS_HOURS": "Add more Days/Hours",
+      "BACK": "Back",
+      "NEXT": "Next",
+      "REMOVE_TEACHER_DATA": "Remove data about the teacher",
+      "SAVE": "Save"
+    },
+    "CONTROL_VALUES": {
+      "FEMALE": "Female",
+      "MALE": "Male",
+      "USE_AVAILABLE_CONTACT_DETAILS": "Use the contact details of the educational services provider"
+    },
+    "HEADERS": {
+      "WORKSHOP_FORM_DESCRIPTION": "workshop, section or study group",
+      "EDIT_WORKSHOP": "WORKSHOP EDITING",
+      "EDIT_WORKSHOP_DESCRIPTION_PATH": "Edit a",
+      "NEW_WORKSHOP": "NEW WORKSHOP",
+      "NEW_WORKSHOP_DESCRIPTION_PATH": "Add a new"
+    },
+    "LABELS": {
+      "AGE_OF_CHILDREN": "Age of children",
```

Comment:
can you please apply labels from this section to the same fields in workshop-details (remove it from there), so we will have only one place with such data and will not support both sections if something will be renamed